theano/tensor/tests/test_basic.py

```diff
@@ -6332,6 +6332,8 @@ def test_var():
     f = function([a], var(a, axis=2))
     assert numpy.allclose(numpy.var(a_val, axis=2), f(a_val))
 
+    f = function([a], var(a, corrected=True))
+    assert numpy.allclose(numpy.var(a_val), f(a_val))
```

Comment:
I don't think numpy.var uses the corrected two pass algorithm so you probably to implement the corrected two path algorithm as well for the test.